helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/constraints/TestInstancePartitionsCountConstraint.java

```diff
@@ -46,18 +46,16 @@ public void testWhenInstanceIsIdle() {
   @Test
   public void testWhenInstanceIsFull() {
     when(_testNode.getAssignedReplicaCount()).thenReturn(10);
-    when(_clusterContext.getEstimatedMaxPartitionCount()).thenReturn(10);
     float score =
         _constraint.getAssignmentNormalizedScore(_testNode, _testReplica, _clusterContext);
-    Assert.assertEquals(score, 0.5f);
+    Assert.assertEquals(score, 0.099667996f);
```

Comment:
Is this the calculation result of f(X) = tanh(1/x)? could we just make a note here so that we have the context later on when we need to fix the test?

Reply:
Yeah, the value looks weird when it appears in the unit test class, thanks I will mark it in the java doc